### PR TITLE
Alert component created. 

### PR DESCRIPTION
**Pull request by borkarsaish65**
image

Added alert component

### CODE REVIEW

File: bootstrap.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+
+        <div style="height: 100px;" class="d-flex">
```
Comment (mohammed-sas) [inline bootstrap.html]: avoid inline styling

File: bootstrap.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    <i class="far fa-clipboard"></i>
+   </button>
+
+   <button type='button' class='btn-floating btn-large'>
+    <i class="far fa-clipboard"></i>
+   </button>
+
+
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
```
Comment (mohammed-sas) [inline bootstrap.html]: what's this script for

File: component/alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+.alert-warning {
+    background-color: #ffb72387;
+    color: #6a0f0feb;
+    border-color: #d3939de8;
+}
+
+.alert-success {
+    background-color: #59eca5d4;
```
Comment (mohammed-sas) [inline component/alert/alert.css:28]: use css variables for colors , put them in main css file

**Review by mohammed-sas — CHANGES_REQUESTED** [review]
suggested changes

File: bootstrap.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+
+
+   <button type='button' class='btn btn-link'>Click</button>
+      
+
+
+
+
+   <button type='button' class='btn btn-primary'>
```
Comment (kart-c) [inline bootstrap.html]: I would suggest to use a code formatter

**Review by kart-c — APPROVED** [review]
I would suggest to work on the changes suggested by @mohammed-sas before merging the branch

File: bootstrap.html (hunk 4 above)
Comment (borkarsaish65) [reply bootstrap.html]: Thanks. Yes making the changes asap.

File: component/alert/alert.css (hunk 3 above)
Comment (borkarsaish65) [reply component/alert/alert.css:28]: Thanks mohammed. Making the changes.

File: bootstrap.html (hunk 2 above)
Comment (borkarsaish65) [reply bootstrap.html]: its not used. I will remove it.

File: bootstrap.html (hunk 1 above)
Comment (borkarsaish65) [reply bootstrap.html]: yes sir.

**Review by mohammed-sas — APPROVED** [review]

**Pull request merged**